api/src/test.rs

```diff
@@ -0,0 +1,637 @@
+// Tests for server.rs api implementations
+
+use crate::server::decode_event_data;
+use crate::server::Server;
+use crate::{AccessInterestStore, AccessModelStore};
+use anyhow::Result;
+use async_trait::async_trait;
+use ceramic_api_server::Api;
+use ceramic_api_server::{
+    models::{self},
+    EventsEventIdGetResponse, EventsPostResponse, InterestsSortKeySortValuePostResponse,
+    SubscribeSortKeySortValueGetResponse,
+};
+use ceramic_core::{Cid, Interest};
+use ceramic_core::{EventId, Network, PeerId, StreamId};
+use mockall::{mock, predicate};
+use multibase::Base;
+use recon::{Key, Sha256a};
+use std::str::FromStr;
+use tracing_test::traced_test;
+
+mod tests {
```

Comment:
you can remove this mod scoping now since the entire file is the tests module. We do not need another nested test module.